### PR TITLE
Removes Goon from Build.js

### PR DESCRIPTION
**Pull request by san7890**
Hey there,

I assume I'm reading the script right. This folder doesn't exist any more on the repository, so we should probably drop it since it is no longer relevant to the function of this script.

I assume my changing of nearly every line in this file is related to Prettier running on the Javascript File. I assume this is the behavior that Prettier is meant to do, please do let me know if I'm fucking something up somehow.

### CODE REVIEW

File: tools/build/build.js [hunk 1]
```diff
@@ -6,107 +6,106 @@
...
 export const DmTarget = new Juke.Target({
   parameters: [DefineParameter],
   dependsOn: ({ get }) => [
-    get(DefineParameter).includes('ALL_MAPS') && DmMapsIncludeTarget,
+    get(DefineParameter).includes("ALL_MAPS") && DmMapsIncludeTarget,
   ],
   inputs: [
-    '_maps/map_files/generic/**',
-    'code/**',
-    'goon/**',
```
Comment (san7890) [inline tools/build/build.js:68]: this is the line that matters btw

**Comment by stylemistake** [comment]
Prettier should be ignoring this file since it doesn't follow the configuration defined in tgui folder, so it formatted everything badly.

If possible, just commit the line that has changed.

**Comment by san7890** [comment]
> Prettier should be ignoring this file since it doesn't follow the configuration defined in tgui folder, so it formatted everything badly.
> 
> If possible, just commit the line that has changed.

Alright, I'll try something.

**Comment by san7890** [comment]
alright i just disabled prettier then recommited

**Pull request merged**